.github/workflows/containers.yml

```diff
@@ -0,0 +1,89 @@
+name: Create containers
+
+on:
+  # run every night
+  schedule:
+    - cron: "0 22 * * *"
+
+  # schedule manually
+  workflow_dispatch:
+    inputs:
+      # On workflow dispatch, `branch` is selected by default
+      # You can access it in `github.ref_name`
+
+      tag_name:
+        description: "Tag name for the container"
+        required: true
+        default: "nightly"
+
+  # TODO: remove
```

Comment:
do it

Reply:
[602fd7b](https://github.com/geo-engine/geoengine-ui/pull/450/commits/602fd7ba7c8c1ba246f01a706858da3cf9185312)